ci/ansible/roles/pulp-certs/tasks/main.yml

```diff
@@ -5,42 +5,48 @@
     name: openssl-perl
     state: present
   when:
-    - ansible_distribution == "Fedora"
+    - ansible_distribution == 'Fedora'
     - ansible_distribution_major_version|int == 26
 
 - name: Create CA to sign all certificates
   command: >
     /usr/bin/openssl req -x509 -newkey rsa:2048 -keyout private/cakey.pem -nodes -days 3650
-     -out cacert.pem -subj '/C=US/ST=North Carolina/L=Raleigh/O=Pulp/OU=Development/CN=PulpCA'
+    -out cacert.pem
+    -subj '/C=US/ST=North Carolina/L=Raleigh/O=Pulp/OU=Development/CN=PulpCA'
   args:
     chdir: /etc/pki/CA/
     creates: cacert.pem
 
 - name: Create CA index.txt
-  file: path=/etc/pki/CA/index.txt state=touch owner=root group=root mode=0644
+  file:
+    path: /etc/pki/CA/index.txt
+    state: touch
+    owner: root
+    group: root
+    mode: 0644
 
 - name: Enable dynamic CA trust store
   shell: update-ca-trust enable
-  when: ansible_distribution == "RedHat" and ansible_distribution_major_version|int == 6
+  when: ansible_distribution == 'RedHat' and ansible_distribution_major_version|int == 6
 
 - name: Add CA to trust store
-  shell: >
-    cp /etc/pki/CA/cacert.pem /etc/pki/ca-trust/source/anchors/cacert.pem
+  shell: cp /etc/pki/CA/cacert.pem /etc/pki/ca-trust/source/anchors/cacert.pem
   args:
     creates: /etc/pki/ca-trust/source/anchors/cacert.pem
 
 - name: Update CA trust
   shell: update-ca-trust
 
 - name: Configure openssl subjectAltName
-  lineinfile: dest=/etc/pki/tls/openssl.cnf
-              insertafter="^\[ usr_cert \]"
-              line="subjectAltName=DNS:{{ ansible_fqdn }},DNS:{{ ansible_nodename }},DNS:{{ ansible_hostname }}"
+  lineinfile:
+    dest: /etc/pki/tls/openssl.cnf
+    insertafter: '^\[ usr_cert \]'
```

Comment:
I recently found out that yaml treats single and double quotes differently. This produces a syntax error:

```
insertafter: "^\[ usr_cert \]"
```